docker-compose.prod.yml

```diff
@@ -24,4 +24,4 @@ services:
             - ./cfgov/apache/conf/:/src/cfgov-refresh/cfgov/apache/conf/
             - ./cfgov/apache/conf.d/:/src/cfgov-refresh/cfgov/apache/conf.d/
             - ./cfgov/apache/conf.modules.d/:/src/cfgov-refresh/cfgov/apache/conf.modules.d/
-        
+        
```

Comment:
Mind reverting the changes to this file for a cleaner history?

Reply:
sure I can do that